src/check.ts

```diff
@@ -59,8 +59,12 @@ async function processSigs() {
         msgHash: true
       }
     };
-    const results = await snapshot.utils.subgraphRequest(subgraphUrl, query);
-    results.sigs.forEach(sig => processSig(sig.account, sig.msgHash));
+    try {
+      const results = await snapshot.utils.subgraphRequest(subgraphUrl, query);
+      results.sigs.forEach(sig => processSig(sig.account, sig.msgHash));
```

Comment:
We should do try catch only with the subgraph request not processSig, this is already handled in the function

Reply:
No, it is not, 

It can still fail if there is a connection issue with the database (SELECT query)
https://github.com/snapshot-labs/snapshot-relayer/blob/5c70e1d14775f8244610f24523fa725e8658eb41/src/check.ts#L30-L41

we need to add more try catches in more places than leaving the server crashed state

Reply:
@bonustrack Added trycatch to only subgraph request, please check again